ssn/rdf/sosa.ttl

```diff
@@ -140,20 +140,22 @@ sosa:Sample
   rdf:type rdfs:Class ;
   rdf:type owl:Class ;
   rdfs:comment "Feature on which observations may be made, which is intended to be representative of a FeatureOfInterest that is not fully accessible."@en ;
+  rdfs:comment "Physical samples are sometimes known as 'specimens'."@en ;
   rdfs:comment """Samples are artifacts of an observational strategy, and have no significant function outside of their role in the observation process. The characteristics of the samples themselves are of little interest, except perhaps to the manager of a sampling campaign.
 
 A Sample is intended to sample some FatureOfInterest in an application domain, so there is an expectation of at least one sampledFeature property. However, in some cases the identity, and even the exact type, of the sampled feature may not be known when observations are made using the sampling features."""@en ;
   rdfs:label "Sample"@en ;
   skos:definition "Feature on which observations may be made, which is intended to be representative of a FeatureOfInterest that is not fully accessible."@en ;
   skos:example "A 'station' is essentially an identifiable locality where a sensor system or Process may be deployed and an observation made. In the context of the observation model, it connotes the 'world in the vicinity of the station', so the observed properties relate to the physical medium at the station, and not to any physical artifact such as a mooring, buoy, benchmark, monument, well, etc."@en ;
+  skos:example "A  statistical sample is often designed to be characteristic of a big population, so that observations can be made regarding the sample that provide a good estimate of the properties of the wider population."@en ;
```

Comment:
I will remove 'big' and 'wide' here and instead will use the term 'population' in its statistical meaning.